docs/philosophy/outcomes/developer_adoption.md

```diff
@@ -176,38 +157,28 @@ It's important with this measurement to capture, not only the time it takes to g
 
 #### Data Points
 
-The following data points must be gathered from the systems we are aiming to measure.
-
-_Adoption trigger ($T$)_
-
-:   Target user requests access
-
-    Expressed as a timestamp
-
-_Adoption events by user ($E$)_
-
-:   A unique target user does something valuable using the tool
-
-    Expressed as a set of timestamps
+[_Adoption events $(E)$_](#adoption-events)
 
 #### Formulas
 
-_Adoption Lead Time ($LT$)_
+_Adoption lead time at a given time $(L(t))$_
 
-:   For any individual adoption event, the adoption lead time $LT$ can be calculated as follows:
+:   Given a timestamp $t$, the adoption lead time for any individual adoption event at time $t$, is calculated as follows:
 
     $$
-    LT = min(E) - T
+    L(t) = \min(\{x:(\text{username},x) \in E\}) - t
     $$
 
+TODO still confused
```

Comment:
After our talk in Slack, I understood Adoption Lead Time. To confirm: Average Adoption Lead Time is only interesting to teams that have multiple adoption events, right?

Reply:
@mateusoliveira43 it's less about teams having multiple adoption events, and more about the assumption that multiple people/teams will adopts the same tool/platform, and Average Adoption Lead Time is the wholistic aggregated measure of the Time to Adopt for the tool

Reply:
oh got it, it is related to target users, I was thinking about adoption events, thanks

Will remove TODO and update the formula to remove my mistake